build_artifacts/v2/v2.9/v2.9.0/dirs/usr/local/bin/start-sagemaker-ui-jupyter-server

```diff
@@ -43,13 +43,17 @@ if [ -n "$SAGEMAKER_RECOVERY_MODE" ]; then
     --ServerApp.base_url="/$SAGEMAKER_APP_TYPE_LOWERCASE/default" \
     --ServerApp.token='' \
     --ServerApp.allow_origin='*' \
-    --SchedulerApp.db_url='sqlite:////tmp/.jupyter_scheduler_do_not_delete.sqlite'
+    --SchedulerApp.db_url='sqlite:////tmp/.jupyter_scheduler_do_not_delete.sqlite' \
```

Comment:
will need to confirm if this works w recovery mode as well, it doesn't have `--collaborative` flag